templates/repo/header.tmpl

```diff
@@ -52,15 +52,15 @@
 						<form method="post" action="{{$.RepoLink}}/action/accept_transfer?redirect_to={{$.RepoLink}}">
 							{{$.CsrfTokenHtml}}
 							<div data-tooltip-content="{{if $.CanUserAcceptTransfer}}{{$.locale.Tr "repo.transfer.accept_desc" $.RepoTransfer.Recipient.DisplayName}}{{else}}{{$.locale.Tr "repo.transfer.no_permission_to_accept"}}{{end}}">
-								<button type="submit" class="ui button {{if $.CanUserAcceptTransfer}}green {{end}} ok inverted small"{{if not $.CanUserAcceptTransfer}} disabled{{end}}>
+								<button type="submit" class="ui button green ok inverted small"{{if not $.CanUserAcceptTransfer}} disabled{{end}}>
```

Comment:
If the `inverted` styles is broken, either:

1. Completely remove them and never use them
2. Fix them

We should choose one , instead of patching them again and again, I have seen too many fixes for `inverted` buttons.

Reply:
Removed `inverted` here in https://github.com/go-gitea/gitea/pull/24510/commits/81c988c61dc47d8bb134fd8fff1cb3e7bc17e6d1 and updated despcription.

Reply:
I mean ... are there still other usages of such `reverted`? Should we clean up them together.

Reply:
No, these are actually the last places using `inverted` in templates, but there are some styles in `semantic.css` defining `inverted` related css rules, which might require rebuilding of semantic if we need to remove them, but i am not sure how.

Reply:
That's good news.

ps: to make the buttons not that prominent, could we use "ui basic green/red button"? These solid backgrounds look a little strange to me (not a must).

Reply:
Sure, `basic` buttons look better, done in https://github.com/go-gitea/gitea/pull/24510/commits/2c680f7b180997b087ea873c9e1f1d07a92908f6